dagfactory/dagbuilder.py

```diff
@@ -217,17 +217,6 @@ def _handle_http_sensor(operator_obj, task_params):
                     headers["Content-Type"] = "application/json"
                 task_params["headers"] = headers
         elif HTTP_SENSOR_CLASS and issubclass(operator_obj, HTTP_SENSOR_CLASS):
-            if not (
-                task_params.get("response_check_name") and task_params.get("response_check_file")
-            ) and not task_params.get("response_check_lambda"):
-                raise DagFactoryException(
-                    "Failed to create task. HttpSensor requires \
-                    `response_check_name` and `response_check_file` parameters \
-                    or `response_check_lambda` parameter."
-                )
-
-            # remove dag-factory specific parameters
-            # Airflow 2.0 doesn't allow these to
```

Comment:
Were we able to test this? I am worried if the else block below might cause a problem for the lambda evaluation. Would be nice to confirm the behavior once end to end with a DAG if not already tested